docs/guides/install-qiskit.mdx

```diff
@@ -126,7 +131,7 @@ Whether you will work locally or in a cloud environment, the first step for all
 If you are planning to work locally and use simulators built into Qiskit, then your installation is done. If you want to run jobs on IBM Quantum systems, next [select an access channel](setup-channel) and finish your setup.
 
 <Admonition type="note" title="Stay current with the latest versions">
-    Periodically check the [Qiskit release notes](../api/qiskit/release-notes) and the [Qiskit Runtime release notes](../api/qiskit-ibm-runtime/release-notes) to see new releases. We recommend frequently updating your requirements for `qiskit` and `qiskit-ibm-runtime` by, for example, changing the versions in `requirements.txt` to the latest versions.
+    Periodically check the [Qiskit release notes](../api/qiskit/release-notes) and the [Qiskit Runtime release notes](../api/qiskit-ibm-runtime/release-notes) to see new releases, then run `pip install --upgrade <package-name>` to upgrade to thew new version. We recommend frequently updating your requirements for `qiskit` and `qiskit-ibm-runtime` by, for example, changing the versions in `requirements.txt` to the latest versions.
```

Comment:
This is not a safe way to upgrade. It won't change the requirements.txt file that everyone else in your project consumes. Real projects should not be upgrading with `pip install --upgrade`.

Reply:
But what about people who are just using their own environment, like when I need to update my own environment?  Do we not expect that to happen?

Reply:
I'll go ahead and make the change.